### ISSUE TITLE
Request blocked by CloudFlare

### ISSUE THREAD

**Issue by raphael-jolimoi** [rbignon/doctoshotgun#307]
Sadly it seems there is  some Cloudfare security. 

I was able to run it after retrying it several times. It seems a bit random.

**Comment by raphael-jolimoi** [comment]
I would add to be careful not to run several times the script in parallel. That might get your IP banned which can be solved by a VPN or changing your connection source. Thanks for this great tool.

**Comment by vdaubry** [comment]
same here :

```
 % ./doctoshotgun.py fr paris ... --only-third --pfizer
Password:
Request blocked by CloudFlare
Traceback (most recent call last):
[...]
woob.browser.exceptions.ServerError: 503 Server Error: Service Temporarily Unavailable
```

**Comment by nicolaus-hee** [comment]
> I was able to run it after retrying it several times. It seems a bit random.

Agree. Just had it on what was definitely the first run in weeks. Worked on the 2nd attempt 1 minute later.
